onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -681,6 +681,11 @@ def reshape_to_1d(tensor):
     return op.SequenceMap(self, body=reshape_to_1d)
 
 
+@torch_op("aten::atleast_1d", trace_only=True)
```

Comment:
Possible to make it not trace only, such that the function name is retained in the graph?

Reply:
Would nested affect performance?